notion/helper.py

```diff
@@ -0,0 +1,66 @@
+import os
+
+from notion.interfaces import INotionAPIClient, INotionDataParser
+
+
+class NotionDataParser(INotionDataParser):
+    def _format_text(self, text, value_type):
+        if value_type == 'text':
+            return text
+        elif value_type == 'bulleted_list':
+            return f"•   {text}"
+        else:
+            return None
+
+    def _parse_text(self, title):
+        text_list = []
+        for text_data in title:
+            if len(text_data) == 1 or (
+                    text_data[1][0] and text_data[1][0][0] == 'a'):
+                text_list.append(text_data[0])
+        complete_text = ''.join(text_list)
+        return complete_text
+
+    def parse(self, data):
+        page = []
+        for _, data_value in data.items():
+            value = data_value.get('value')
+            value_type = value.get('type')
+            title = value.get('properties', {}).get('title', [['']])
+            text = self._parse_text(title)
+
+            if not text:
+                break
```

Comment:
Should this be break or continue, just asking, not going deep into logic

Reply:
Its break;